src/Compilers/VisualBasic/Portable/Lowering/LambdaRewriter/LambdaRewriter.vb

```diff
@@ -681,7 +694,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Dim frame As LambdaFrame = Nothing
```

Comment:
The initializer 'Nothing' is not used on any control path and looks unnecessary.